src/briefcase/bootstraps/pygame.py

```diff
@@ -8,6 +8,7 @@ def app_source(self):
         return """\
 import importlib.metadata
 import os
+import pkg_resources
```

Comment:
pkg_resources is a [deprecated API](https://setuptools.pypa.io/en/latest/pkg_resources.html) (and is setuptools specific to boot); preference is to use `importlib.metadata`.

Reply:
Thanks; I can't keep track of what in packaging is deprecated and what's not...